index.html

```diff
@@ -61,12 +92,12 @@ <h1 class="projects">
                     Projects
                    </h1>
                      <div class="projectlist">
-                    <img class="laptop" src="./images/Rectangle21.png" alt="laptop">
-                    <div class="rectangle">
-                    <p class="first-project"> First Project </p>
+                    <img class="laptop1" src="./images/Rectangle21.png" alt="laptop">
```

Comment:
Please align your content properly carefully inspect the design you picked on Figma.

Your design:

![Screenshot from 2022-07-15 22-22-23](https://user-images.githubusercontent.com/79506105/179288677-d68a5965-6a9c-4132-bb29-c8f52d93870a.png)

Figma design:

![Screenshot from 2022-07-15 22-20-58](https://user-images.githubusercontent.com/79506105/179288818-07e6236c-318a-469a-9a5e-9ecd369d86f2.png)

Please pay attention to detail.